README.md

```diff
@@ -1 +1,190 @@
-# open-days-registration-application-front-end
\ No newline at end of file
+# General description
+
+This repository has been created due the open sourcing process of the 2019 Open Days ticket reservation system. The repository contains two Front-end projects written in [Angular](https://angular.io/) using [Angular Material design](https://material.angular.io/) and scripts for building a portable [Docker](https://www.docker.com/) container, which hosts the two applications using Apache httpd. 
```

Comment:
> This repository has been created due the open sourcing process of the 2019 Open Days ticket reservation system
I think that it is not necessary to mention explicitly that we are `open sourcing` the code. I would remove it

Reply:
fixed